src/routes/guides/signatures/+page.svelte

```diff
@@ -0,0 +1,157 @@
+    <li>Remove the header from the code</li>
+    <li>Base45 decode the code</li>
+    <li>
+      Parse the <em>alg</em> and <em>kid</em> from the COSE header, as well as the COSE payload
+      <strong>without verifying the signature</strong>
+    </li>
+    <li>
+      Fetch <em>https://{"{"}payload.iss{"}"}/.well-known/product-passport.json</em>
+      <a href="/tags/metadata/#product-passport.json">metadata</a>
+    </li>
+    <li>
+      Fetch the <a href="/tags/metadata/#jwks.json">JWKS keys</a> from <em>{"{"}jwks_url{"}"}</em> in
+      the product-passport.json
+    </li>
+    <li>Verify the COSE signature matches the key provided via JWKS</li>
+    <li>
+      Fetch <em
+        >https://{"{"}payload.iss{"}"}/.well-known/product-passport/{"{"}payload.product{"}"}.json</em
+      >
+      for <a href="/tags/metadata/#%7Bproduct%7D.json">product metadata</a>
+    </li>
+    <li>
+      Fetch <em
+        >https://{"{"}product_dataspace{"}"}.well-known/dataspace/dataspace-configuration.json</em
+      >
+    </li>
+    <li>
+      Proceed to do what you wish with the <em>supported_dataproducts</em> from the
+      <a href="/tags/metadata/#product-passport.json">{"{"}payload.product{"}"}.json</a>, including
+      e.g. fetching the data products via the <em>product_gateway_url</em> from the
+      <em>dataspace-configuration.json</em>
+    </li>
+  </ol>
+
+  <p>
+    Check out our Sandbox's <A
+      href="https://sandbox.ioxio-dataspace.com/.well-known/dataspace/dataspace-configuration.json"
+      >dataspace-configuration.json</A
+    >
+    to see what configuration is available for IOXIO dataspaces.
+  </p>
+
+  <SectionTitle title="Example signed code" />
+  <p>
+    The line feeds are added to make it easier to see what the code is like, but are not a part of
+    the code. Having extra linefeeds breaks the code.
+  </p>
+
+  <Code lang={json}>
+    {`
+IT1:RRQ5 8/60V500GKOG8WA6E+5OPCZQEG/D5ECBPE ED0AFM2E6VCQ/EV9EV3E $E9WEB$CBECP9EY CPED$.
+CKWEZED.2DV50 NF19W:5EKV52W0TFQJ01Y%8W+6%9IRE2SJ51VEU95K:AM+61XB2UJ+/81-I0Y3YXABQ9XJQST
+CRB7G3Y%D868%$M0DE E30QNSFL64W/V473PWW0. VLK8BJO8VBTEHUDQS9HX3BCOMRFFDE560C.BHJP0000DP$
+YPCBNQ2OPPA8XD/N5Z6LNVNEJM* CL8FH7PHRUDVVO5IDEQZ$SM1U-N8U$UZ0KQPBYOSEN4: P2/7+5NECC254I
+C7BZCN.0J%BBEOMD645V2RG7+S0.5CCR2JTCCAVVOVN0OT-K63CSR14TARR%N/27AUQ9XNJMVO-0LV5DSJPH2/O
+V6MMZ1FLLG17WGDYP51T8W/06:QV43FK7RPC$-1Q+PK%I
+`}
+  </Code>
+
+  <p>Printed into a QR code one might look something like this:</p>
+
+  <img alt="example of a signed IOXIO Tag QR code" src="/ioxio-tag-example-signed.png" />
+
+  <p>
+    Check out the <em>make_cose_code</em> function in <A
+      href="https://github.com/ioxiocom/ioxio-tags/blob/main/api/app/tag.py">our demo application</A
+    > for a full example.
+  </p>
+
+  <p>
+    You can check out our <A href="https://generator.tags.ioxio.dev">IOXIO Tag&trade; generator</A> and
+    as well as <A href="https://scanner.tags.ioxio.dev">IOXIO Tag&trade; scanner</A>, and their <A
```

Comment:
Looks like there's both an `and` and an `as well as` and one of them seems a bit extra.
```suggestion
    You can check out our <A href="https://generator.tags.ioxio.dev">IOXIO Tag&trade; generator</A> as
    well as <A href="https://scanner.tags.ioxio.dev">IOXIO Tag&trade; scanner</A>, and their <A
```